dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -340,6 +340,7 @@ public static void restore() throws Exception{
     public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent1And2_ViewSpanishContent2And3(
             final Container container, final Template template) throws Exception{
 
+        Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", true);
```

Comment:
really this set is not doing nothing because the next line is setting this same property to false